stdlib/_typeshed/__init__.pyi

```diff
@@ -10,6 +10,31 @@ from os import PathLike
 from typing import AbstractSet, Any, Container, Generic, Iterable, Protocol, TypeVar, Union
 from typing_extensions import Final, Literal, final
 
+_WU = TypeVar("_WU")
+
+class WeakUnion(Generic[_WU], Any):
```

Comment:
```suggestion
class WeakUnion(Any, Generic[_WU]):
```

I'm curious to see if it makes any difference if we have `Any` first instead of second